### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by veggiemonk**
It is time to evolve a little bit, what do you think team?

**DON'T MERGE UNTIL MOST OF US HAVE SEEN THIS PR**

### CODE REVIEW

**Comment by devhkr** [comment]
Thanks @veggiemonk for updating the guideline

Some remarks / personal opinions:
- Shouldn't a 'Mission Statement' in the first place, before 'I want to share a project what should I do'?. I mean it's a short introduction what the `awesome-docker` is about and what kind of projects we're looking for. Then, a 'How to contribute' can be the next section.
- As far as I know, we rejected some blogs (PR #352 , #276 etc.), meaning that we should mention that in the guideline. Unless, it's an advanced well documented tutorial which can be accepted to the list.

**Comment by veggiemonk** [comment]
@devhkr good points

**Comment by veggiemonk** [comment]
@devhkr thanks for your comments. 
Updated

**Comment by veggiemonk** [comment]
**DON'T MERGE UNTIL MOST OF US HAVE SEEN THIS PR**

**Comment by devhkr** [comment]
Beside fixing typos, I made some changes.
Especially removing redundancies.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,28 +43,27 @@ But usually, we are pretty relaxed people, so just come and say hi, we'll figure
...
 - Description should be clear and concise (read it out loud to be sure)
 - Description should follow the link, on the same line
-- if you want to add more than one link, please don't do all PR on the exact same line, it usually results in conflicts and your PR cannot be automatically merged...
+- If you want to add more than one link, please don't do all PR on the exact same line, it usually results in conflicts and your PR cannot be automatically merged...
 
 Please contribute links to packages/projects you have used or are familiar with. This will help ensure high-quality entries.
 
 
-#### Your commit message will be a tweet so make a beautiful commit message, keep that in mind :)
+#### Your commit message will be a [tweet](https://twitter.com/awesome_docker) so write a [good commit message](https://chris.beams.io/posts/git-commit/), keep that in mind :)
```
Comment (veggiemonk) [inline CONTRIBUTING.md]: hehe nice! 👍

**Comment by veggiemonk** [comment]
@devhkr Much better! Thanks :D

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,71 +4,77 @@ Please note that this project is released with a [Contributor Code of Conduct](h
...
-- **To add to the list:** Submit a pull request or open an issue
-- **To remove from the list:** Submit a pull request or open an issue
-- want to change something: Submit a pull request or open an issue
-- want to try something else: Submit a pull request or open an issue
-- don't know what to do: Submit a pull request or open an issue, let us know what's going on.
+# Quality Standards
 
-or you can
+Note that we can help you achieve those standards, just try your best and be brave.
+We'll guide you to the best of our abilities.
```
Comment (dmitrytokarev) [inline CONTRIBUTING.md]: :heart:

**Comment by dmitrytokarev** [comment]
@veggiemonk do you think it's worth mentioning that projects should preferably be open source and if they are not they should at least have an option for a free account/trial period? WDYT?

**Comment by devhkr** [comment]
@dmitrytokarev On the pull request template, there is one rule which states:

`MONEY: indicates if one needs a paying subscription to use that project you could just add ($$$) next to the name ([my project - $$$] (link to my project) )`

In addition, nowadays most paid services are offering free account/trial period.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,71 +4,77 @@ Please note that this project is released with a [Contributor Code of Conduct](h
...
+- [Contribution Guidelines](#contribution-guidelines)
+- [New Collaborators](#new-collaborators)
 
-# Contribution Guidelines
+# Mission Statement
 
-#### Your commit message will be a tweet so make a beautiful commit message, keep that in mind :)
+`awesome-docker` is a hand-crafted list for high-quality information about Docker and its resources. It should be related or compatible with Docker. If it's just an image built on top of Docker, the project possibly belongs to other [awesome lists](https://github.com/sindresorhus/awesome). You can check the [awesome-selfhosted list](https://github.com/Kickball/awesome-selfhosted) or the [awesome-sysadmin list](https://github.com/n1trux/awesome-sysadmin) as well.
+If it's a **tutorial or a blog post**, they get outdated really quickly so we don't really put them on the list but if it is on a very advanced and/or specific topic, we will consider it!
+If something is awesome, share it (pull request or issue or chat), let us know why and we will help you!
```
Comment (gesellix) [inline CONTRIBUTING.md]: Should we add a link for "chat" to gitter?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,71 +4,77 @@ Please note that this project is released with a [Contributor Code of Conduct](h
...
+But usually, we are pretty relaxed people, so just come and say hi, we'll figure something out together.
+
+# Contribution Guidelines
+
+## I want to share a project, what should I do?
+
+- **Adding to the list:** Submit a pull request or open an issue
+- **Removing from the list:** Submit a pull request or open an issue
+- Changing something else: Submit a pull request or open an issue
+- Don't know what to do: Open an issue or join our chat, let us know what's going on.
```
Comment (gesellix) [inline CONTRIBUTING.md]: Again: I would prefer to link "chat" to gitter.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -4,71 +4,77 @@ Please note that this project is released with a [Contributor Code of Conduct](h
...
+
+### Rules for Pull Request
```
Comment (gesellix) [inline CONTRIBUTING.md]: nit: s/Request/Requests/

**Comment by dmitrytokarev** [comment]
Another idea i just saw here: https://github.com/vinta/awesome-python#contributing
is to leave some decision making to the community - if the PR gets certain number of thumbups then we merge - but then you have to deal with the community...

**Comment by Moshe-Immerman** [comment]
Just a couple ideas, mainly stealing from https://github.com/akullpp/awesome-java/blob/master/CONTRIBUTING.md

Under Quality Standards:

- Self-promotion is frowned upon and viewed critically, but your suggestion will of course be approved if the criteria match.

And under Rules for Pull Request:

- Use the following format for libraries: [LIBRARY](LINK) - DESCRIPTION.
- Link to the github repository not the project page e.g. Use *https://github.com/moby/moby*
 instead of *https://docker.com*
- Entries should be sorted in ascending alphabetical order, i.e. a to z.
- End all descriptions with a full stop/period.

**Comment by devhkr** [comment]
Thanks @Moshe-Immerman for the idea.

I would rather say: **Self-promotion on behalf of an organization will be reviewed critically**.
Personally, self-promotion (speaking of personal projects) is to be encouraged for being innovative and making great projects.

The rules for pull request can be added to the `pull_request_template.md` which I'm going to update.

**Comment by devhkr** [comment]
Sounds interesting @dmitrytokarev 

Currently, it's better to leave the decision to the maintainers as we haven't had much problem yet with decision-making. Maybe we can consider it for later.

File: CONTRIBUTING.md (hunk 4 above)
Comment (devhkr) [inline CONTRIBUTING.md]: Thanks for the hint @gesellix 
Originally it was there right after this text.
Just updated.

**Comment by veggiemonk** [comment]
@devhkr @agebhar1 @dmitrytokarev @Moshe-Immerman @gesellix @vegasbrianc well done guys!!!

**Pull request merged**